lib/mobius/services/ets_shelf.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Mobius.Services.ETSShelf do
+  @moduledoc false
+
+  use GenServer
+
+  @timeout 1_000
+
+  @spec start_link(keyword) :: GenServer.on_start()
+  def start_link(opts) do
+    GenServer.start_link(__MODULE__, opts, name: __MODULE__)
+  end
+
+  @spec create_table(atom, [atom | tuple]) :: :ok | {:error, :ets_error}
+  def create_table(name, opts) when is_atom(name) do
+    # Creates the table or gives it back if the server already owns it
```

Comment:
```suggestion
    # Creates the table or gives it back if the shelf already owns it
```
Server is a pretty vague term